src/SvgArrow.js

```diff
@@ -33,6 +49,23 @@ function computeEndingArrowDirectionVector(endingAnchorOrientation) {
   }
 }
 
+export function computeStartingPointAccordingToArrowHead(
+  xArrowHeadStart: number,
+  yArrowHeadStart: number,
+  arrowLength: number,
+  strokeWidth: number,
+  startingAnchorOrientation: AnchorPositionType,
+) {
+  const startingVector = computeStartingArrowDirectionVector(startingAnchorOrientation);
+
+  const { arrowX, arrowY } = startingVector;
+
+  const xStart = xArrowHeadStart + (arrowX * arrowLength * strokeWidth) / 2;
+  const yStart = yArrowHeadStart + (arrowY * arrowLength * strokeWidth) / 2;
+
+  return { xStart, yStart };
+}
```

Comment:
Looks like this whole logic is duplicated, right? Renaming the functions so that they don't depend on `end` or `start` would've been enough to reuse it.

You can leave it like this, I'll refactor this myself :wink:

Reply:
I initially duplicated them as I thought I would need them later, but forgot to clean up when I made the PR.
Should be ok now.